src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    private ColorSupplier colorSupplier;
+    private Random random;
+
+    public FigureSupplier() {
+        colorSupplier = new ColorSupplier();
+        random = new Random();
+    }
+
+    public Figure getRandomFigure() {
+        String color = colorSupplier.getRandomColor();
+        int figureType = random.nextInt(5);
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/532ca3b6-04d1-4fa7-a6f1-a7964b2369a7)

Reply:
Not fixed.
5 should be constant